paderbox/io/csv_module.py

```diff
@@ -1,11 +1,35 @@
 
-def load_csv(file, sniffer=False, sniffer_sample_length=1024):
+
+def load_csv(
+        file,
+        *,
+        fieldnames=None,
+        dialect="excel",
+        sniffer=False,
+        sniffer_sample_length=1024,
+):
     """
     Load a csv file to python.
 
     This function should imitate pandas.read_csv, but should return instead
     of a pandas Dataframe a list of dicts.
 
+    Args:
+        file:
+        fieldnames:
+            When a csv has no header, you can manually specify the fieldnames.
+            Note, when fieldnames is given, it is assumed, that the csv has no
+            header.
+        dialect:
+            e.g. 'excel', 'excel-tab' and 'unix'.
+            See python documentation for standard library csv for more details.
+        sniffer:
+            Whether to detect the dialect (i.e. delimiter and line separator)
+            When True, ignore dialect.
+        sniffer_sample_length:
+
+    Returns:
+        list of dicts
```

Comment:
I prefer to have the type annotations in the function definition, e.g.,
```python
def load_csv(
        file,
        *,
        fieldnames: Optional[List[str]]=None,
        dialect: str="excel",
        sniffer: bool=False,
        sniffer_sample_length: int=1024,
) -> List[dict]:
```
But either one is fine

Reply:
done